Example/URBNSwiftAlert/ExampleViewController.swift

```diff
@@ -7,18 +7,393 @@
+    }
+    
+    var longMessage: String {
+        return "And the message that is a bunch of text that will turn scrollable once the text view runs out of space.\nAnd the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text.  And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text."
+    }
+    
+    var customView: UIView {
+        let customView = UIView()
+        customView.backgroundColor = .green
+        let iv = UIImageView(image: UIImage(named: "IMG_7016"))
+        iv.widthAnchor.constraint(equalToConstant: 100).isActive = true
+        iv.heightAnchor.constraint(equalToConstant: 100).isActive = true
+        iv.contentMode = .scaleAspectFit
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.text = "Cat In Box, you can tap inside to dismiss."
+        let sv = UIStackView(arrangedSubviews: [iv, label])
+        sv.spacing = 20
+        sv.wrap(in: customView, with: InsetConstraints(insets: UIEdgeInsets(top: 20, left: 20, bottom: 20, right: 20), priority: UILayoutPriorityDefaultHigh))
```

Comment:
updated the name of the image from JPG to jpg, for some reason Xcode cares about that, can you see if this image shows up now?